src/ffi/c.rs

```diff
@@ -75,54 +87,63 @@ impl Drop for StreamWrapper {
     }
 }
 
-const ALIGN: usize = std::mem::align_of::<usize>();
+#[cfg(all(feature = "any_zlib", not(feature = "libz-rs-sys")))]
+mod allocator {
+    use super::*;
 
-fn align_up(size: usize, align: usize) -> usize {
-    (size + align - 1) & !(align - 1)
-}
+    use std::alloc::{self, Layout};
+    use std::convert::TryFrom;
+    use std::os::raw::c_void;
+
+    const ALIGN: usize = std::mem::align_of::<usize>();
+
+    fn align_up(size: usize, align: usize) -> usize {
+        (size + align - 1) & !(align - 1)
+    }
+
+    pub extern "C" fn zalloc(_ptr: *mut c_void, items: uInt, item_size: uInt) -> *mut c_void {
```

Comment:
What's the motivation for the change from `AllocSize` to `uInt`?

I see this was previously set to `pub type AllocSize = libz::uInt;`, and the `*` imports don't make it easier to see where the type is actually coming from.

Reply:
yeah, `AllocSize` was only used in this signature (turns out) and I didn't think that the alias really had any value at that point.